### ISSUE TITLE
nested values groupValue doesn't work

### ISSUE THREAD

**Issue by knownasilya** [yapplabs/ember-radio-button#3]
The following allows both radio buttons to be toggled at the same time.

```hbs
{{#radio-button value="commercial" groupValue=questions.leaseTo}}
  Commercial
{{/radio-button}}

{{#radio-button value="residential" groupValue=questions.leaseTo}}
  Residential
{{/radio-button}}
```

when `groupValue=leaseTo`, everything works fine.

**Comment by raycohen** [comment]
I tried to create a simple example and it worked as expected.  Can you create a jsbin example of the issue?

**Comment by raycohen** [comment]
@knownasilya can you try this again with recently merged 1.0.0 functionality?

**Comment by knownasilya** [comment]
@raycohen seems to be working
